### ISSUE TITLE
Wrong CSV parsing

### ISSUE THREAD

**Issue by kam1sh** [yandex/ClickHouse#1037]
I have **CSV file** in format like this:
> root@e8c7a78c8c91:/# xzcat /opt/output.csv.xz | head -n1
> 192.168.99.5:8080,42.185,200,4864,1C+Enterprise,1500953104,1500953104,GET,/bitrix/admin/1c_exchange_new.php,47.41,0.0

but when i send it to clickhouse, he tries to parse field **DateTime** next the second Date field and fails:

> root@e8c7a78c8c91:/# xz -v -c -d < /opt/output.csv.xz | clickhouse-client --host=clickhouse-server --query="INSERT INTO web.logs FORMAT CSV";
> Code: 27. DB::Exception: Cannot parse input: expected , before: rix/admin/1c_exchange_new.php,47.41,0.0\r\n92.63.110.113:8080,31.932,200,4864,1C+Enterprise/8.3,1500953104,1500953104,GET,/bitrix/admin/1c_exchange_new.php,62.63,: (at row 1)
> 
> Row 1:
> Column 0,   name: Host,        type: String,   parsed text: "192.168.99.5:8080"
> Column 1,   name: Time,        type: Float64,  parsed text: "42.185"
> Column 2,   name: Status,      type: UInt16,   parsed text: "200"
> Column 3,   name: Size,        type: UInt32,   parsed text: "4864"
> Column 4,   name: UserAgent,   type: String,   parsed text: "1C+Enterprise"
> Column 5,   name: date,        type: Date,     parsed text: "1500953104"
> Column 6,   name: Timestamp,   type: DateTime, parsed text: "1500953104,GET,/bit"
> ERROR: garbage after DateTime: "rix/admin/"
> ERROR: DateTime must be in YYYY-MM-DD hh:mm:ss or NNNNNNNNNN (unix timestamp, exactly 10 digits) format.

What am i doing wrong? I thought, the problem in CSV format, but ClickHouse parses all other fields fine. And problem not in tiimestamp field, the fifth column with the same number parses fine.
If it makes sense, i'm using your Docker images (yandex/clickhouse-server and client)

**Comment by kam1sh** [comment]
Also, i formatted this Timestamp field into "YYYY-MM-DD hh:mm:ss", and after that CSV was successfully eaten by ClickHouse. But it's temporary solution.

**Comment by kam1sh** [comment]
There is also weird things with date field parsing. ClickHouse parsed it without errors, but when i executed `SELECT max(date), min(date) FROM logs` it gave me 0000-00-00 in both columns.

But when i dropped table and created it from another CSV, where this field was not in timestamp but in "2017-07-25", `max(date), min(date)` gave me the expected date fields that were in CSV (2017-07-26 and 2017-07-25).

**Comment by alexey-milovidov** [comment]
Parsing from unix timestamp is not supported for Date fields for now.

**Comment by ztlpn** [comment]
@ludv1x fixed DateTime CSV parsing in #1098 
The fix is available in 1.1.54276 release.

**Comment by ztlpn** [comment]
Sorry for misinformation - the fix was not included in the 1.1.54276 release. It is in master now and will be included in the next release.

**Comment by et2010** [comment]
I have the same issue.

```
Code: 117. DB::Exception: Expected end of line: (at row 16453)

Row 16452:
Column 0,   name: SYS_AVG_Current_I_U,    type: Int64,    parsed text: "342"
...
Column 137, name: t_204,                  type: Float64,  parsed text: "-8.8"
Column 138, name: t_216,                  type: Float64,  parsed text: "-7.4"
Column 139, name: timestamp,              type: DateTime, parsed text: "1548987920"
ERROR: garbage after DateTime: "4"
ERROR: DateTime must be in YYYY-MM-DD hh:mm:ss or NNNNNNNNNN (unix timestamp, exactly 10 digits) format.
```
with
```
ClickHouse client version 19.13.1.11 (official build).
```

Any suggestions?

**Comment by filimonov** [comment]
@et2010 maybe related to #6426
It was already fixed in 19.11.3
Try to update.
